app/src/main/java/com/github/h3lp3rs/h3lp/notification/NotificationService.kt

```diff
@@ -0,0 +1,121 @@
+package com.github.h3lp3rs.h3lp.notification
+
+import android.annotation.SuppressLint
+import android.app.Notification
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.graphics.BitmapFactory
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import com.github.h3lp3rs.h3lp.R
+
+/**
+ *  Static class containing useful function to throw notification.
+ *  The notification are stylized with H3LP app style
+ *  @warning allways make sure to use {@link createNotificationChannel} method to make sure the notification channel is initialized
```

Comment:
Maybe just add a line return, this comment is a bit longer than the usual limit for line size